### DISCUSSION TITLE
AWS CloudFront distribution S3 bucket access configuration

### DISCUSSION THREAD

**Discussion by miguelduque** [serverless/serverless#10569] [Q&A]
I'm deploying a AWS Cloudfront distribution with the serverless-api-cloudfront plugin.

This distribution has a S3 origin configured which works as expected.

Configuration:

    plugins:
    - serverless-api-cloudfront
    
    mywebapp:
       component: "@sls-next/serverless-component@3.6.0"
       inputs:
          env:
             ...
          cloudfront:
             defaults: # options for lambda that handle SSR
                ...
             origins:
                - url: https://...
                  S3OriginConfig:
                      OriginAccessIdentity:
                  pathPatterns:
                      secure/exports/*:
                        minTTL: 10
                        maxTTL: 10
                        defaultTTL: 10
                        forward:
                          cookies: "none"
                          queryString: true
        ...

My problem is that When my distribution is depoyed, the S3 bucket access configuration of my S3 origin is always set to "Yes use OAI (bucket can restrict access to only CloudFront)" and I would like it to be "Don't use OAI (bucket must allow public access)".

https://imgur.com/6Aagn0T

Any idea on how to configure this?

**Comment by pgrzesik** [comment]
Hello @miguelduque - I think you might have more luck finding answer to this question here: https://github.com/serverless-nextjs/serverless-next.js as this is not the repository for component-based deployments.

**Reply by miguelduque** [reply]
Thank you, I will do that
